devops/ansible/ssl.yml

```diff
@@ -0,0 +1,39 @@
+- name: Install certbot
+  apt:
+    name: certbot
+    update_cache: yes
+  become: true
+
+- name: Copy post cert renewal hook script
+  copy:
+    src: arango-ssl.sh
+    dest: /etc/letsencrypt/renewal-hooks/deploy/
+    mode: u=rwx,g=,o=
+  become: true
+
+- name: Set server name inside of the post cert renewal hook script
+  replace:
+    path: /etc/letsencrypt/renewal-hooks/deploy/arango-ssl.sh
+    regexp: '(SERVER_NAME)'
+    replace: '{{ server_name }}'
+  become: true
+
+- name: Generate a certificate
+  shell:
+    cmd: certbot certonly -d {{ server_name }} --standalone -m jwilburn@sci.utah.edu --agree-tos -n
+  become: true
+
+- name: Run arango-ssl.sh (doesn't run above since it's not a renewal)
+  shell:
+    cmd: /etc/letsencrypt/renewal-hooks/deploy/arango-ssl.sh
+  become: true
+
+- name: Add cronjob for cert renewal
+  cron:
+    name: certbot renew
+    minute: "0"
+    hour: "0,12"
+    job: "python -c 'import random; import time; time.sleep(random.random() * 3600)' && certbot renew -q"
```

Comment:
What's the randomized delay for?

Reply:
This is from the certbot site. It's so the letsencrypt server doesn't get crushed at 00:00 and 12:00. It makes a random delay before running the renew command. This code is directly from the old version of the docs (it was removed when they chose to prefer snaps)